netbox_topology_views/migrations/0009_disable_physics.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.2.10 on 2024-04-29 14:37
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('netbox_topology_views', '0007_individualoptions_group_locations_and_more'),
```

Comment:
This is problematic, because another PR lists this 0007 as a dependency. This one is no. 0009 and should depend on 0008.

Reply:
Indeed, this happened because I was working on this PR and #488 in parallel. #488 contains a migration with index 0008, therefore this got 0009 and cannot depend on 0008, since it is on another branch.

I propose leaving this comment open until either #488 or this PR gets accepted and change it accordingly as a last step.